packages/backend/src/storage.ts

```diff
@@ -11,7 +14,19 @@ export default (db: IDatabase) => {
 
     //TODO remove
     router.use(cors());
-    router.param('ensName', auth);
+    router.param(
+        'ensName',
+        (req: Request, res: Response, next: NextFunction, ensName: string) => {
+            auth(
+                req,
+                res,
+                next,
+                ensName,
+                req.app.locals.db,
```

Comment:
use the dependencies passed into the function in line 12 instead of req.app.locals.

Reply:
done

Reply:
done